shoop/admin/modules/products/views/edit.py

```diff
@@ -83,6 +83,14 @@ def form_valid(self, form):
                 continue
             if not shop_product_form.instance.pk:
                 shop_product_form.instance.product = self.object
+
+            original_quantity = shop_product_form.instance.minimum_purchase_quantity
```

Comment:
So this change deprecates the rounded_minimum_purchase_quantity property, yeah?

Should the PR mark it as Deprecated.